primitives/Cargo.toml

```diff
@@ -1,15 +1,18 @@
 [dependencies]
 arbitrary = { default-features = false, optional = true, version = "1.0" }
-frame-support = { branch = "moonbeam-polkadot-v0.9.19", default-features = false, git = "https://github.com/purestake/substrate" }
-frame-system = { branch = "moonbeam-polkadot-v0.9.19", default-features = false, git = "https://github.com/purestake/substrate" }
-orml-currencies = { branch = "2022-06-v0.9.19", default-features = false, git = "https://github.com/zeitgeistpm/open-runtime-module-library" }
-orml-tokens = { branch = "2022-06-v0.9.19", default-features = false, git = "https://github.com/zeitgeistpm/open-runtime-module-library" }
-orml-traits = { branch = "2022-06-v0.9.19", default-features = false, git = "https://github.com/zeitgeistpm/open-runtime-module-library" }
-parity-scale-codec = { default-features = false, features = ["derive", "max-encoded-len"], version = "3.0.0" }
+frame-support = { branch = "polkadot-v0.9.23", default-features = false, git = "https://github.com/paritytech/substrate" }
+frame-system = { branch = "polkadot-v0.9.23", default-features = false, git = "https://github.com/paritytech/substrate" }
+orml-currencies = { branch = "polkadot-v0.9.23", default-features = false, git = "https://github.com/open-web3-stack/open-runtime-module-library" }
+orml-tokens = { branch = "polkadot-v0.9.23", default-features = false, git = "https://github.com/open-web3-stack/open-runtime-module-library" }
+orml-traits = { branch = "polkadot-v0.9.23", default-features = false, git = "https://github.com/open-web3-stack/open-runtime-module-library" }
+parity-scale-codec = { default-features = false, features = [
+    "derive",
```

Comment:
can you please update to taplo 0.7.0 and then run format script? New version avoids this kind of formatting.

Reply:
I updated to taplo 0.7.0 and run the script, but it does not apply any changes. Do I have to revert all of that now and format again or is there another way?

Reply:
unfortunately it seems so.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/811/commits/a350f05fd64af01cd41b38f9216f9ad20a3f0c51